### DISCUSSION TITLE
Set tabwidth for verbatim block

### DISCUSSION THREAD

**Discussion by xade93** [zim-desktop-wiki/zim-desktop-wiki#2084] [Q&A]
Hi community, the default tabwidth for verbatim text is 8. Is is possible to change it in the settings? Thanks :D

**Comment by jaap-karssenberg** [comment]
Verbatim is just a formatting style, so tab size will apply same as in the rest of the buffer. See "GUI style config file" at https://zim-wiki.org/manual/Help/Config_Files.html how to configure.

**Comment by xade93** [comment]
Sorry, I just tried this yesterday, but tabwidth in particular seems to be ignored by Zim...
The style.conf is respected by Zim, as a modification to the color of verbatim text will be reflected; but regardless of how large I set indent, the tab is visually displayed as 8 characters.

Also, should I set "indent size in pixels" anyways? since it's monospace, shouldn't I instead set tabwidth as the number of characters it should display? Idk

Attached is the style.conf snippet. Thanks
```
[Tag code]
foreground=#1a192b
family=monospace
indent=4

[Tag pre]
foreground=#1a192b
family=monospace
wrap-mode=GTK_WRAP_NONE
indent=4
```

**Reply by introt** [reply]
Hi, I turned this into a question for better visibility - hope that's alright!

**Comment by ihavenomouth** [comment]
It seems to be a bug, the tabs property is ignored in the style.conf file.

**Comment by dgasaway** [comment]
The value of `indent` under `[Tag pre]` sets how much the block is indented, not the tab size.  You can set `tabs` under `[TextView]` but you'll have to use # of pixels rather than # of characters.
